MediaGraph/topologies/evr-motion-files-assets/readme.md

```diff
@@ -1,8 +1,8 @@
-# Event-based video recording to local files and Assets based on motion events
+# Event-based video recording to local files and assets based on motion events
```

Comment:
recording to assets and local files....

That way it's clear that "local" applies to files, and not local "files and assets"...

Reply:
changed